pages/linux/wami.md

```diff
@@ -0,0 +1,20 @@
+# wami
+
+> An open-source and easy-to-use tool that recommends suitable programs for for specific tasks.
+> More information: <https://github.com/evait-security/wami>.
+
+- Find expanded results in all categories from the lake and [S]sort them in the specified order:
```

Comment:
```suggestion
- Find expanded results in all categories from the lake and [S]ort them in the specified order:
```
There seems to be a small typo, after this the PR is GTG.

Reply:
Thank you, changes are applied.